atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -87,7 +87,10 @@ def _get_registries(self):
         """
         Return a list of registries that this build updated
         """
-        return self.workflow.push_conf.all_registries
+        registries = self.workflow.push_conf.pulp_registries
+        if not registries:
+            registries = self.workflow.push_conf.all_registries
+        return registries
```

Comment:
This is wrong in general. We needed to change this to return all registries for the benefit of delete_from_registry, see https://github.com/projectatomic/atomic-reactor/commit/e73808a5.

Reply:
Oh, that makes sense. In this case a PR to delete manifest lists from registry would resolve that.

Renaming this to `[WIP]` until this is implemented